rllib/utils/metrics/stats.py

```diff
@@ -368,26 +386,28 @@ class for details on the reduction logic applied to the values list, based on
             reduced = reduced[0]
 
         if not compile and not self._inf_window:
-            return self._numpy_if_necessary(self.values).copy()
+            if reduced_values is None:
+                _, reduced_values = self._reduced_values()
```

Comment:
This happens if we don't have new values, don't want to compile and use a window.
I.e. when we reduce windowed stats on leaf MetricsLoggers.